docs/bsp.md

```diff
@@ -279,26 +301,26 @@ trait BuildServerCapabilities {
   
   /** The server can provide a list of targets that contain a
     * single text document via the method textDocument/buildTargets */
-  def textDocumentBuildTargetsProvider: Boolean
+  def providesTextDocumentBuildTargets: Boolean
```

Comment:
I'm concerned of diverging from LSP here on naming conventions. I would prefer to keep the name and turn `Boolean` into an object, like LSP does.

If we keep this change, we should synchronize the name with `canTest`/`canRun`/`canCompile`.

Reply:
I changed it because I didn't know LSP used `provider` when the type of the field was `boolean`, I agree with you that we should follow the convention there so I'll revert this change.